src/components/ArticlePreview/ArticlePreview.scss

```
@@ -0,0 +1,94 @@
+@use 'src/styles/mixins' as m;
+@use 'src/styles/variables' as v;
+
+.article-list {
```

Let's add more specificity as our selectors should be unique now
```suggestion
.article-preview-list {
```